themes/grav/templates/forms/fields/editor/editor.html.twig

```diff
@@ -5,8 +5,10 @@
 
 {% block label %}
     {% if field.label %}
-        {% set hint = field.help ? 'data-hint="' ~ field.help|tu|raw ~ '"': '' %}
-        <div class="form-label form-field hint--bottom" {{ hint }}>{{ field.label|tu|raw }}</div>
+        <div
+          class="form-label form-field hint--bottom"
+          {% if field.help %}data-hint="{{field.help|tu|raw}}" {% endif %}
+        >{{ field.label|tu|raw }}</div>
```

Comment:
You should try `|e('html_attr')` instead of `|raw` (but keep the `|tu` for translation.

```yaml
{% if field.help %}data-hint="{{field.help|tu|e('html_attr')}}" {% endif %}
```